config/routes.rb

```diff
@@ -62,6 +62,9 @@
     end
     resources :stories_bulk_destroy, only: [:create]
     resources :stories_bulk_update, only: [:create]
+    namespace :stories do
+      post '/bulk_create', action: :create, controller: 'bulk_create'
+    end
```

Comment:
Follow the convention of the other routes:

```suggestion
    resources :stories_bulk_create, only: [:create]
```